src/lesson2/lesson2.java

```diff
@@ -0,0 +1,50 @@
+package lesson2;
+
+public class lesson2 {
+    public static void main(String[] args) {
+        compareDiapazon(10, 20);
+        isPositivePrint(10);
+        System.out.println(isNegative(-10));
+        printString("Hello world!",10);
+        System.out.println(isLeapYear(1501));
+    }
+
+    private static boolean compareDiapazon(int a, int b) {
+        int result = a + b;
+        if (result >= 10 && result <= 20) return true;
+        else return false;
+    }
+
+    private static void isPositivePrint(int i) {
+        if ((i >= 0)) {
+            System.out.println("Число " +  i + " положительное");
+        } else {
+            System.out.println("Число " +  i + " отрицательное");
+        }
+    }
+
+    private static boolean isNegative(int i) {
+        if ((i < 0)) {
```

Comment:
внутренние скобки избыточны
можно упростить:
```suggestion
        return i < 0;
```